features/step_definitions/sign_in_steps.rb

```diff
@@ -109,6 +109,14 @@ def make_accounts(role, number = 1)
   sign_in(@case_worker.user, 'password')
 end
 
+Given('a case worker admin user account exists') do
+  @case_worker = create(:case_worker, :admin)
+end
+
+When('I sign in as a case worker admin') do
```

Comment:
```suggestion
When('I sign in as a case worker') do
```
this was me  - the step is not specific to a caseworker admin sign in but to the current `@case_worker` instance, and should therefore reflect that